tests/ssg_test_suite/oscap.py

```diff
@@ -526,12 +528,19 @@ def _find_rule_result_in_output(self):
         return match[-1]
 
     def _analyze_output_of_oscap_call(self):
-        local_success = True
+        local_success = 1
         # check expected result
         rule_result = self._find_rule_result_in_output()
 
+        if rule_result == "notapplicable":
+            msg = (
+                'Rule {0} evaluation resulted in {1}'
+                .format(self.rule_id, rule_result))
+            LogHelper.preload_log(logging.WARNING, msg, 'notapplicable')
```

Comment:
@jan-cerny I can remove this message, so instead of
```
INFO - xccdf_org.ssgproject.content_rule_dconf_gnome_screensaver_lock_enabled
WARNING - Script correct_value_unlocked.fail.sh using profile xccdf_org.ssgproject.content_profile_pci-dss notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_dconf_gnome_screensaver_lock_enabled evaluation resulted in notapplicable
WARNING - Rule xccdf_org.ssgproject.content_rule_dconf_gnome_screensaver_lock_enabled evaluation resulted in notapplicable
```
we would only see
```
INFO - xccdf_org.ssgproject.content_rule_dconf_gnome_screensaver_lock_enabled
WARNING - Script correct_value_unlocked.fail.sh using profile xccdf_org.ssgproject.content_profile_pci-dss notapplicable
```

Reply:
Maybe it would be better to rather skip remediation phase in case initial phase reports notapplicable. I will try to update it.

Reply:
Should be resolved by f00b3af